frontend/console/src/features/timeline/TimelineCronScheduled.tsx

```diff
@@ -0,0 +1,17 @@
+import type { CronScheduledEvent } from '../../protos/xyz/block/ftl/v1/console/console_pb'
+import { formatTimestampShort } from '../../utils/date.utils.ts'
+import { verbRefString } from '../verbs/verb.utils'
+
+export const TimelineCronScheduled = ({ cron }: { cron: CronScheduledEvent }) => {
```

Comment:
We can do it in a follow on PR, but we'll also want and need a `TimelineCronDetails.tsx` component for the right side bar.

And we'll need to update the verb page event list to show these correctly (seems like it has the `call` icon).
![Screenshot 2024-10-01 at 3 36 15 PM](https://github.com/user-attachments/assets/46846671-fef9-4282-a739-20b0acb48e1e)